src/styles/index.scss

```diff
@@ -1,3 +1,95 @@
 body {
   margin: 0;
 }
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-style: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__price {
+    font-size: 12px;
+    font-weight: 400;
+    height: 18px;
+    color: #616070;
+  }
+
+  &__number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    text-align: right;
+    margin-top: -34px;
+    margin-left: 85px;
+  }
+
+  &__button {
+    background-color: #00acdc;
+    color: #fff;
+    cursor: pointer;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+    line-height: 16px;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    gap: 17px;
+    color: #060b35;
+  }
+
+  .stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  .stars__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  .stars--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The SCSS for the `stars` block should be placed in a separate file as per the task requirements. Ensure that this block is moved to its own SCSS file.